Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache-2.0"
 
 [dependencies]
 rocket = { git = "https://github.com/SergioBenitez/Rocket", rev = "8d4d01106e2e10b08100805d40bfa19a7357e900" }
-ipfs-embed = { git = "https://github.com/ipfs-rust/ipfs-embed", rev = "ba82076d4950c31f974425827635bfd0adb28c62" }
+ipfs-embed = { git = "https://github.com/ipfs-rust/ipfs-embed", rev = "ba82076d4950c31f974425827635bfd0adb28c62", default-features = false, features = ["tokio"] }
 anyhow = "1.0"
 ssi = { git = "https://github.com/spruceid/ssi", rev = "df98fabf6b86a816a0a2fb22b191f49f95e0a952", features = ["secp256k1", "secp256r1", "ripemd160", "keccak-hash"] }
 tokio = { version = "1", features = ["rt", "macros", "rt-multi-thread"] }
```

Comment:
Is it possible to only use the tokio re-exported by Rocket? To avoid any versioning problem in the future.

Reply:
thats actually semi how IPFS-embed handles it now, it will only work when called in the context of a user-started runtime (which is started by Rocket in this case), although it does bundle a tokio dependancy, that should be de-duplicated by cargo by resolving to a mutually compatible version.